package-lock.json

```diff
@@ -1,3326 +0,0 @@
-{
```

Comment:
Why are you removing this? Can you explain this change.

Reply:
Package lock is malware unless you are making unpublished to registry project like bot

Reply:
And why is it 'malware'?

Reply:
locks user out from easy version switching for library used packages